Library/.rubocop.yml

```diff
@@ -6,6 +6,12 @@ AllCops:
     - '**/Casks/**/*'
     - '**/vendor/**/*'
 
+require:
+  - ./Homebrew/rubocops/bottle_block_cop.rb
```

Comment:
@GauthamGoli it's failing because of this error:

```
Error: obsolete parameter IndentWhenRelativeTo (for Style/CaseIndentation) found in /Users/alyssa/code/brew/Library/.rubocop.yml
`IndentWhenRelativeTo` has been renamed to `EnforcedStyle`
obsolete parameter AlignWith (for Lint/EndAlignment) found in /Users/alyssa/code/brew/Library/.rubocop.yml
`AlignWith` has been renamed to `EnforcedStyleAlignWith`
```